docs/12-design.md

```diff
@@ -137,7 +136,7 @@ make any difference to the results.
 
 ## State store
 
-The state store for a table holds information about the files that are currently in the table,
+The state store for tables holds information about the files that are currently in each table,
```

Comment:
A state store is still for a single table conceptually, and in terms of how we talk about it. The fact that they can share the same underlying store is specific to the implementation, while a table's state store can be implemented differently to another table's.